safe/gui/tools/minimum_needs/needs_calculator_dialog.py

```diff
@@ -22,14 +22,14 @@
 from PyQt4 import QtGui
 from PyQt4.QtCore import pyqtSignature, pyqtSlot, QSettings
 
-from safe.common.utilities import temp_dir
+from safe.common.utilities import temp_dir, unique_filename
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->